server/datastore/mysql/migrations/tables/20251222174712_AddHostLocationSupport.go

```diff
@@ -0,0 +1,32 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20251222174712, Down_20251222174712)
+}
+
+func Up_20251222174712(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+	CREATE TABLE host_locations (
```

Comment:
`host_last_known_locations`? Table name sounds more like a location track until you notice the unique key being on host ID.